test/integration/associations/scope.test.js

```diff
@@ -11,6 +11,10 @@ const current = Support.sequelize;
 const Op = Sequelize.Op;
 const semver = require('semver');
 
+const {
```

Comment:
Why did you add this here? It caused problems with some of the other dialects and does not seem to be used in this file

Reply:
The only thing I can think of is that it was some sort of debug tool that didn't get cleaned up. Removing now